### PR TITLE
Sprint report for Sprint 19

### PR DESCRIPTION
**Pull request by bgeuken**

### CODE REVIEW

File: _posts/development/2017-05-05-frontend-sprint-report-1.html [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 layout: post
-title: Highlights of the OBS frontend development sprint
+title: Highlights of the OBS frontend development – Sprint 15
```
Comment (hennevogel) [inline _posts/development/2017-05-05-frontend-sprint-report-1.html]: *nitpick* is the sprint number really of any value for people reading this?

Comment (bgeuken) [inline _posts/development/2017-05-05-frontend-sprint-report-1.html]: I think it is. Especially when you look at the overview page: http://openbuildservice.org/blog/

Comment (bgeuken) [inline _posts/development/2017-05-05-frontend-sprint-report-1.html]: And it doesn't hurt

**Comment by bgeuken** [comment]
merged now. We can change remove the sprint numbers later, if we want to.

**Pull request merged**